<?xml version="1.0" ?>
<launch xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="https://gist.githubusercontent.com/nalt/dfa2abc9d2e3ae4feb82ca5608090387/raw/roslaunch.xsd">
    <arg name="simulation_mode" default="false"/>

    <group ns="cpu_interface">
        <rosparam file="$(find cpu_interface)/config/default.yaml" command="load"/>  
        <param name="simulation_mode" value="$(arg simulation_mode)"/>
    </group>

</launch>
